cogs/assistance-cmds/dnsmitm.switch.md

```diff
@@ -0,0 +1,8 @@
+---
+title: DNS.mitm
+help-desc: Applying AMS DNS redirection
+---
+
+To ensure that your emuMMC/emuNAND does not connect to Nintendo, you should use DNS.mitm (AMS DNS redirection).
+
+[Follow these steps](https://nh-server.github.io/switch-guide/extras/blocking_updates/) to set up DNS.mitm on your emuMMC/emuNAND and redirect requests to Nintendo.
```

Comment:
"...redirect requests to Nintendo."

This is unclear as we're not redirecting the requests to Nintendo. Maybe something like "...redirect requests directed to Nintendo to your SD card/to nothing instead"?

Reply:
Used the wording of the guide